pulseguardian/model/binding.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from sqlalchemy import Column, ForeignKey, Integer, String
+
+from pulseguardian.model.base import Base
+
+
+class Binding(Base):
+    __tablename__ = 'bindings'
+
+    id = Column(Integer, primary_key=True)
+    exchange = Column(String(255))
+    routing_key = Column(String(255))
+    queue_name = Column(String(255), ForeignKey('queues.name'))
```

Comment:
I admit I'm not fully up to date with the best practices here, but would it not be better to have a foreign key to the queue id (e.g. reference the queue itself, instead of just the name)?